modules/cli/src/main/scala/scala/cli/commands/Package.scala

```diff
@@ -152,14 +162,24 @@ object Package extends ScalaCommand[PackageOptions] {
       else destPath.toString
 
     def alreadyExistsCheck(): Unit =
-      if (!force && os.exists(destPath)) {
+      if (!force && os.exists(destPath) && expectedModifyEpochSecondOpt.isEmpty) {
         System.err.println(
           s"Error: $printableDest already exists. Pass -f or --force to force erasing it."
         )
         sys.exit(1)
       }
```

Comment:
Maybe we can merge both checks, like
```suggestion
      val alreadyExists = !force &&
        os.exists(destPath) &&
        expectedModifyEpochSecondOpt.forall(exp => os.mtime(destPath) != exp)
      if (alreadyExists) {
          val msg =
            if (expectedModifyEpochSecondOpt.isEmpty) s"$printableDest already exists"
            else s"$printableDest was overwritten by another process"
        System.err.println(s"Error: $msg. Pass -f or --force to force erasing it.")
        sys.exit(1)
      }
```